automated install/basic-install.sh

```diff
@@ -1638,23 +1647,23 @@ installPiholeWeb() {
     local str="Creating directory for blocking page, and copying files"
     printf "  %b %s..." "${INFO}" "${str}"
     # Install the directory
-    install -d /var/www/html/pihole
+    install -d -m 0755 $PI_HOLE_DASHBOARD_DIR
```

Comment:
Please use the `${VAR}` form of expansion.

Reply:
Ok, fixed.